### PR TITLE
TAARlite GUID Ranking job

### PR DESCRIPTION
**Pull request by crankycoder**
The TAARlite guid ranking job was accidentally removed during the rewrite of many jobs.

This adds it back in.

The job successfully generated the JSON file at `s3://telemetry-parquet/taar/lite/guid_install_ranking.json` which contains a map of GUIDs to installation counts

```
$ aws s3 ls s3://telemetry-parquet/taar/lite/guid_install_ranking.json
2020-09-02 15:22:53    6194266 guid_install_ranking.json
```

### CODE REVIEW

**Comment by crankycoder** [comment]
If you check the commit history for the taarlite guid ranking job - the old job used clients_daily - this new version just uses bigquery to get the (addon_guid, install_count) tuple - other than that - there are no significant changes.

File: dags/taar_daily.py [hunk 1]
```diff
@@ -218,6 +218,70 @@
...
+        worker_disk_type="pd-ssd",
+        master_disk_size=1024,
+        worker_disk_size=1024,
+        master_num_local_ssds=2,
+        worker_num_local_ssds=2,
+    ),
+    dag=dag,
+)
+
+taar_delete_requests = SubDagOperator(
```
Comment (haroldwoo) [inline dags/taar_daily.py:257]: does taar_delete_requests get called? not seeing it

File: dags/taar_daily.py [hunk 2]
```diff
@@ -218,6 +218,70 @@
...
+            "--date",
+            "{{ ds }}",
+            "--aws_access_key_id",
+            taar_aws_access_key,
+            "--aws_secret_access_key",
+            taar_aws_secret_key,
+        ],
+        aws_conn_id=taar_aws_conn_id,
+        gcp_conn_id=taar_gcpdataproc_conn_id,
+        master_disk_type="pd-ssd",
```
Comment (haroldwoo) [inline dags/taar_daily.py:247]: maybe comment how this is to make the job run faster

**Review by haroldwoo — APPROVED** [review]
gl hf

File: dags/taar_daily.py (hunk 2 above)
Comment (crankycoder) [reply dags/taar_daily.py:251]: i added comments in now

**Comment by crankycoder** [comment]
closing this as it's just getting ancient

**Comment by eu9ene** [comment]
@crankycoder I'm reviewing your unmerged PRs right now and wanted to merge this one actually...

**Pull request closed without merging**